__tests__/utils/result.utils.test.js

```diff
@@ -0,0 +1,165 @@
+import {
+  STORAGE_RESULTS_BOARD,
+  STORAGE_SCORE_LEVEL_MOOD,
+} from "../../src/constants/constants"
+import { Labels } from "../../src/constants/specificLabels"
+import {
+  convertResultsInStorageToContentTable,
+  getEpdsResultsAndMoodInStorage,
+} from "../../src/utils/result.utils"
+
+describe("Result Utils", () => {
+  describe("getEpdsResultsAndMoodInStorage", () => {
+    beforeAll(() => {
+      global.localStorage = {
+        setItem(key, item) {
+          this.state[key] = item
+        },
+        getItem(key) {
+          return this.state[key]
+        },
+      }
+    })
+
+    test("LocalStorage vide", () => {
+      const expected = {
+        detailQuestions: null,
+        detailReponses: null,
+        moodLabel: undefined,
+      }
+      expect(getEpdsResultsAndMoodInStorage()).toEqual(expected)
+    })
+
+    test("mood et tableau stocké dans le LocalStorage", () => {
```

Comment:
`should return matching questions, responses and mood label when mood and table are stored in local storage`